src/api/clients/IOClients/infra/VBase.ts

```diff
@@ -21,16 +22,32 @@ export class VBase extends InfraClient {
     })
   }
 
+  // Resolve a specific pages-graphql conflict
+  public resolveConflict = (bucketKey: string, path: string, content: any) => {
+    const data = [
+      {
+        op: 'replace',
+        path,
+        value: content,
+      },
+    ]
+
+    return this.http.patch(routes.Conflicts(`vtex.pages-graphql/${bucketKey}`), data, {
+      metric: 'vbase-resolve-conflicts',
+    })
+  }
+
+  // List all conflicts in the pages-graphql bucket
+  public getConflicts = async () => {
+    return this.http.get(routes.Conflicts('vtex.pages-graphql/userData'), {
+      metric: 'vbase-get-conflicts',
+    })
+  }
+
+  // Verify if there is at least one conlfict in the pages-graphql bucket
   public checkForConflicts = async () => {
-    let status: number
-    try {
-      const response = await this.http.get(routes.File('vtex.pages-graphql', 'store/content.json'), {
-        metric: 'vbase-conflict',
-      })
-      status = response.status
-    } catch (error) {
-      status = error.response && error.response.status
-    }
-    return status === 409
+    const response = await this.getConflicts().then(res => res.data)
+
+    return response?.data?.length > 0
```

Comment:
If I am not wrong, axios throws an error for 4xx responses. If that is the case, I think we are changing the behavior here, since the line 49 will raise the error and will not get to the return statement. Have you checked that? Also, as in the other PR, I would avoid the `await / .then` combination.